pnpm-workspace.yaml

```diff
@@ -0,0 +1,3 @@
+packages:
+  - 'server'
+  - 'client'
```

Comment:
`packages` defines the root our [`pnpm` workspace](). `packages` is an array of directories to be included in the workspace.

We're using the `server` and `client` directories here, but this could also be a glob (like `packages/*` which would include all sub-directories of the `packages` directory).